File: Difficulty: Easy/Second Largest/second-largest.py
class Solution:
    def getSecondLargest(self, arr):
        # Code Here
        largest=-1
        smallest=-1
        for i in arr:
            if(i>largest):
                
                smallest=largest
                largest=i
                
            elif(i<largest and i>smallest):
                smallest=i
        return smallest

File: Difficulty: Easy/Second Largest/second-largest.java
class Solution {
    public int getSecondLargest(int[] arr) {
        // code here
        int largest=Integer.MIN_VALUE;
        int sec_largest=Integer.MIN_VALUE;
        for(int i=0;i<arr.length;i++)
        {
            if(arr[i]>largest)
            {
                sec_largest=largest;
                largest=arr[i];
            }
            if(arr[i]<largest && sec_largest<arr[i])
            {
                sec_largest=arr[i];
                
            }
        }
        if (sec_largest == Integer.MIN_VALUE) {
            return -1;
        }
        return sec_largest;
    }
}